exercises/querying-workflows/solution/queryclient.py

```diff
@@ -0,0 +1,13 @@
+import asyncio
+
+from temporalio.client import Client
+from workflow import MyWorkflow
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_workflow_handle("queries")
```

Comment:
```suggestion
    handle = client.get_workflow_handle_for(MyWorkflow.run, "queries")
```

Encourage the type safe form